src/polytopes/segment.jl

```diff
@@ -22,6 +22,9 @@ isclosed(::Type{<:Segment}) = false
 isparametrized(::Type{<:Segment}) = true
 
 nvertices(::Type{<:Segment}) = 2
+nvertices(s::Segment) = nvertices(typeof(s))
```

Comment:
Maybe we should replace the fallback definition for Polytope now that most vertices are static. Can you add this implementation of nvertices to Polytope and erase it from here and from Ngon?